rules/build_electricity.smk

```diff
@@ -163,11 +163,11 @@ if config["enable"].get("build_cutout", False):
             regions_onshore=resources("regions_onshore.geojson"),
             regions_offshore=resources("regions_offshore.geojson"),
         output:
-            protected(CDIR + "{cutout}.nc"),
+            protected(CDIR.joinpath("{cutout}.nc").as_posix()),
```

Comment:
```suggestion
            protected(CDIR / "{cutout}.nc"),
```

Instead of `joinpath`, you should be able to use the` / ` operator (simplifies code)

`.as_posix()` should also not be necessary as

```

In [8]: from pathlib import Path

In [9]: Path("test") / "mypath"
Out[9]: PosixPath('test/mypath')

In [10]: Path("test").joinpath("mypath")
Out[10]: PosixPath('test/mypath')

In [11]: Path("") / Path("")
Out[11]: PosixPath('.')

In [12]: Path("")
Out[12]: PosixPath('.')

In [13]: "" / Path("test")
Out[13]: PosixPath('test')
```
This minimises code changes in `.smk` files to changing "+" to "/".